src/olympia/constants/promoted.py

```diff
@@ -18,6 +18,8 @@
         'badged',
         'autograph_signing_states',
         'can_primary_hero',
+        'subscription',
```

Comment:
Shouldn't this be something like `require_subscription`

Reply:
`subscription` would be fine... though I see we've used that attribute name a lot already so yeah, it's confusing.  https://github.com/mozilla/addons-server/pull/15822/commits/8a03ccd787ff685e6e1bd14eabafa872e59fe21b